spec/host.rb

```diff
@@ -36,15 +36,15 @@
         @hostgroupid2 = zbx.hostgroups.create(:name => gen_name('hostgroup'))
         host = gen_name('host')
         hostid = zbx.hosts.create(
-          host: host,
-          interfaces: [{ type: 1, main: 1, ip: '192.168.0.1', dns: 'server.example.org', port: 10050, useip: 0 }],
-          groups: [
-            {groupid: @hostgroupid},
-            {groupid: @hostgroupid2}
+          :host => host,
+          :interfaces => [{ :type => 1, :main => 1, :ip => '192.168.0.1', :dns => 'server.example.org', :port => 10050, :useip => 0 }],
```

Comment:
2 reasons:
1. stick to single format all across the code
2. the JavaScript style is incompatible with Ruby 1.8.x used in puppet 3.x